core/src/utils/input-masking/types/mask-interface.ts

```diff
@@ -43,8 +43,17 @@ export type MaskPostprocessor = (elementState: ElementState, initialElementState
 
 export interface MaskOptions {
   readonly mask: MaskExpression | ((elementState: ElementState) => MaskExpression);
+  /**
```

Comment:
The `preprocessor`, `postprocessor` and `overwriteMode` are considered internal APIs at this time. They are public API of `maskito`, but we did not agree/design out these options with input masking in our design document. 

I opted for leaving the internal API surface because:
1. It is very minimal (the functions act as noop, returning the existing mask value/selection).
2. These APIs offer fairly complex mask functionality that we likely want to discuss exposing in future iterations. 

As our mask implementation is refined (taking the best from existing solutions + applying our API requirements), this API surface may be removed if it ends up being unneeded or introducing undesired complexity.